### PR TITLE
docs: gitops-apps cn

### PR DESCRIPTION
**Pull request by aFlyBird0**
Signed-off-by: Bird <aflybird0@gmail.com>

## Pre-Checklist

Note: please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/devstream-io/devstream/blob/main/CONTRIBUTING.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [ ] I have added relevant tests

### CODE REVIEW

File: docs/best-practices/apps.zh.md [hunk 1]
```diff
@@ -1,3 +1,153 @@
 # 使用DevStream的新功能Apps来进行GitOps
 
-TODO
+## 0 目标
+
+在本教程中，我们会使用 DevStream 的新特性 应用（Apps），来达到与 [GitOps] 相似的效果。但它的配置更短，来展示 应用 的强大能力。如果你还没有读过原始的 GitOps 最佳实践，可以先点击前面的链接。
```
Comment (daniel-hutao) [inline docs/best-practices/apps.zh.md:5]: link lost

File: docs/best-practices/apps.zh.md [hunk 2]
```diff
@@ -1,3 +1,153 @@
 # 使用DevStream的新功能Apps来进行GitOps
```
Comment (daniel-hutao) [inline docs/best-practices/apps.zh.md:1]: 1. 英文左右的空格
2. “进行GitOps”似乎不通顺；改成“实现”？

**Review by daniel-hutao — CHANGES_REQUESTED** [review]

**Comment by aFlyBird0** [comment]
> This doc looks like a quickstart of apps, maybe you can rename this doc and add config to `internal/pkg/show/config/templates/` to support showing this config.

done in https://github.com/devstream-io/devstream/pull/1356/files ,this pr is just a translation

**Comment by aFlyBird0** [comment]
> This doc looks like a quickstart of apps, maybe you can rename this doc and add config to `internal/pkg/show/config/templates/` to support showing this config.

done in https://github.com/devstream-io/devstream/pull/1356/files ,this pr is just a translation

File: docs/best-practices/apps.zh.md (hunk 2 above)
Comment (aFlyBird0) [reply docs/best-practices/apps.zh.md:1]: 嗯嗯我改一下，这是之前的标题，我就没注意看😂

**Review by daniel-hutao — APPROVED** [review]

**Pull request merged**